### PR TITLE
fix: withPermissions directive was not retrieving the permissions

### PR DESCRIPTION
**Pull request by tiago-freire**
#### What problem is this solving?

"withPermissions" directive was not retrieving the permissions without app name on method parameter of storefront permissions client

#### How to test it?

Use the impersonate feature on a public domain of a B2B store. Example: https://titantools.vtexdemostores.com/

At a myvtex.com domain works because the impersonateB2BUser mutation resolver function does not consider the permissions if adminUserAuthToken exists: https://github.com/vtex-apps/b2b-organizations-graphql/blob/d8e889c7ef75b583047631f835e5b24b6cc332ec/node/resolvers/Mutations/Users.ts#L179

Toast error message:
image

Error on browser console:
`Error: GraphQL error: operation-not-permitted`

Workspace where changes works:
[[Workspace](https://titantools.vtexdemostores.com/#?workspace=dev)]

### CODE REVIEW

**Comment by tiago-freire** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [x]  Patch (backwards-compatible bug fixes)
> * [ ]  Minor (backwards-compatible functionality)
> * [ ]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞

File: node/resolvers/directives/withPermissions.ts [hunk 1]
```diff
@@ -9,7 +9,9 @@ import type StorefrontPermissions from '../../clients/storefrontPermissions'
 export const getUserPermission = async (
   storefrontPermissions: StorefrontPermissions
 ) => {
-  const result = await storefrontPermissions.checkUserPermission()
+  const result = await storefrontPermissions.checkUserPermission(
+    'vtex.b2b-organizations@1.x'
```
Comment (Rudge) [inline node/resolvers/directives/withPermissions.ts:13]: Do you test with the app name as `vtex.b2b-organizations-graphql@0.x`?

**Comment by tiago-freire** [comment]
@Rudge Do I need to do anything else?

**Comment by tiago-freire** [comment]
Bumped by #154

**Pull request closed without merging**